src/kontrol/prove.py

```diff
@@ -244,11 +244,12 @@ def _run_cfg_group(
     summary_ids: Iterable[str],
 ) -> list[APRProof]:
     def init_and_run_proof(test: FoundryTest) -> APRFailureInfo | Exception | None:
+        proof = None
         if Proof.proof_data_exists(test.id, foundry.proofs_dir):
-            apr_proof = foundry.get_apr_proof(test.id)
-            if apr_proof.passed:
+            proof = foundry.get_apr_proof(test.id)
+            if proof.passed:
                 return None
-        start_time = time.time()
+        start_time = time.time() if proof is None or proof.status == ProofStatus.PENDING else 0.0
```

Comment:
Would it make sense to make it `None` instead of `0.0`, and have 
```
  if start_time is not None:
      end_time = time.time()
```
later?

Reply:
Yes, it is more readable. Addressed :)